gallery/gallery/pubspec.yaml

```diff
@@ -35,6 +36,7 @@ dev_dependencies:
 
 flutter:
   assets:
+    - fonts/google_fonts/
```

Comment:
Also I see all the fonts are still included in the directory, is that how it's supposed to work?

Reply:
Yes, see above